TthCrstl/config.ini

```diff
@@ -1 +1,34 @@
-humStart=260
\ No newline at end of file
+# TthCrstl config.ini
+
+# For full descriptions of what each of these entries does,
+# along with the default values for each, see the Wiki page here:
+# https://github.com/profezzorn/ProffieOS/wiki/Sound-Font-config.ini-Configuration
+
+
+humStart=260
+
```

Comment:
no need for blank lines between the variables

Reply:
All fixed.